changelog/std-digest-crc64.dd

```diff
@@ -0,0 +1,16 @@
+Added support for 64 bit CRC
+
+Support for both ISO and ECMA 64 bit CRC was added to $(MREF std, digest, crc).
+
+-------
+import std.digest.crc;
+
+void main()
+{
+    ubyte[8] hash64ecma = crc64ECMAOf("abc");
```

Comment:
I personally perfer the `"abc".crc64ECMAOf` UFCS-style as it reads more naturally in docs, but I really doesn't matter to discuss this for a changelog entry.